control/init.sh

```diff
@@ -1,12 +1,18 @@
-#! /bin/bash
+# This script must be run from BASH with "source control/init.sh".
 
-if [[ $_ == $0 ]]
+if [ "$_" = "$0" ]
```

Comment:
Makes this `if` statement POSIX compliant so it doesn't error if you're not in BASH.